manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -1,5 +1,29 @@
-# Dedicated cluster role - istiod will use fewer dangerous permissions ( secret access in particular ).
-# TODO: separate cluster role with the minimal set of permissions needed for a 'tenant' Istiod
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Also the file is named 'clusterrole.yaml'...

As a namespace admin ( installing istiod ) you can create regular roles AFAIK. This helps with running istio in other namespaces - without having to grant cluster roles or re-run base ( with istio getting configs via XDS-MCP)